docs/configure-rest.md

```diff
@@ -260,3 +260,80 @@ the missing data won't be exported.
 #### `export_options`
 
 Refer [Export Options](configure-rest.md#export_options)
+
+## Private CLI
+
+The ONTAP private CLI allows for more granular control and access to non-public counters. It can be used to fill gaps in the REST API, especially in cases where certain data is not yet available through the REST API. Harvest's REST collectors support the use of the private CLI to address these gaps. This means that even if certain data cannot be collected via the Public REST API, it can still be collected using the private CLI.
+
+For more information on using the ONTAP private CLI with the REST API, you can refer to the following resources:
+
+- [NetApp Documentation: Accessing ONTAP CLI through REST APIs](https://docs.netapp.com/us-en/ontap-automation/rest/access_ontap_cli.html)
+- [NetApp Blog: Private CLI Passthrough with ONTAP REST API](https://netapp.io/2020/11/09/private-cli-passthrough-ontap-rest-api/)
+
+
+### Creating Templates with Private CLI
+
+Let's take an example of how a CLI command:
+
+```bash
+system fru-check show
+```
+
+REST APIs endpoint:
+
+```http
+/api/private/cli/system/fru-check?fields=node,fru_name,fru_status
+```
+
+In this example, the CLI command `system fru-check show` gets converted to the API endpoint with a forward slash for every word prefixing `/api/private/cli/`.
+
+The `show` command gets converted to the HTTP method GET call. From the CLI, look at the required field names and pass them as a comma-separated value in `fields=` in the API endpoint.
+
+Note: If the field name contains a hyphen (-), convert it to an underscore in the REST API field. For example, `fru-name` → `fru_name`.
+
+### Advanced and Diagnostic Mode Commands
+
+CLI Pass through allows you to execute even advanced and diagnostic mode CLI commands with just one setting in the API endpoint. You should add `privilege_level=diagnostic` or `privilege_level=advanced` under the filter setting.
```

Comment:
The CLI pass through allows you to execute advanced and diagnostic mode CLI commands by including the `privilege_level` field in your request under the filter setting like so: